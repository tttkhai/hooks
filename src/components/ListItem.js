import React from 'react'

export default function ListItem({id, description}) {
    return (
        <div> id: {id}. Description: {description} </div>
    )
}

