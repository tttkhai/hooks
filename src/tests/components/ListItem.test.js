import { shallow } from 'enzyme';
import React from 'react';
import ListItem from '../../components/ListItem';
import expenses from '../fixtures/text'

test('test ListItem component snapshot', ()=>{
    const wrapper = shallow(<ListItem {...expenses[0]} />)
    expect(wrapper).toMatchSnapshot()
})