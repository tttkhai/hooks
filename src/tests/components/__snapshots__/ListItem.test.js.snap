// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ListItem component snapshot 1`] = `
<div>
   id: 
  1
  . Description: 
  Hello
   
</div>
`;
